frontend/src/components/Home/components/UserHomePage.js

```diff
@@ -19,10 +19,19 @@ const UserHomePage = ({ isAuthenticated }) => {
     }
   };
 
-  useEffect(updateProfileFromServer, []);
+  useEffect(() => updateProfileFromServer(), []);
 
   const { home } = ROUTES;
+
+  // TODO: to be refactored
+  const onProfileUpdated = profileData => {
+    setUserProfile(profileData);
+  };
+  home["account"].props = {
```

Comment:
Any reason we don't use dot notation here?
```suggestion
  home.account.props = {
```

Reply:
Not really.